packages/phoenix-client/src/phoenix/client/resources/experiments/evaluators.py

```diff
@@ -10,6 +11,64 @@
 
 if TYPE_CHECKING:
     from phoenix.client.resources.experiments.types import Evaluator
+    from phoenix.evals.preview.evaluators import Evaluator as EvalsEvaluator
+    from phoenix.evals.preview.evaluators import Score as EvalsScore
+
+    ScoreResult = Union["EvalsScore", list["EvalsScore"]]
+
+
+@functools.cache
+def _evals_installed() -> bool:
+    return importlib.util.find_spec("phoenix.evals") is not None
+
+
+def is_evals_evaluator(obj: Any) -> bool:
+    def _is_duck_typed_evaluator(obj: Any) -> bool:
+        # for forwards compatibility
+        # TODO: delete this once evals 2.0 is out of preview
+        has_bind = hasattr(obj, "bind")
+        has_direction = hasattr(obj, "direction")
+        has_schema = hasattr(obj, "input_schema")
+        has_source = hasattr(obj, "source")
+        return all([has_bind, has_direction, has_schema, has_source])
+
+    if not _evals_installed():
+        return False
+
+    try:
+        from phoenix.evals.preview.evaluators import Evaluator as EvalsEvaluator
```

Comment:
not sure I follow why it makes sense to put this in before we are out of preview given this signature will change.